packages/core/misc-widgets/src/components/github-star/GithubStar.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div
+    class="kong-ui-public-misc-widgets-github-star"
+    data-testid="github-star"
+  >
+    <a
+      aria-label="Star buttons/github-buttons on GitHub"
+      class="github-button"
+      data-color-scheme="no-preference: light; light: light; dark: light;"
+      data-show-count="true"
+      :href="url"
```

Comment:
suggestion: even though the GitHub script typically takes care of opening the link, can we add a couple extra attributes to ensure it always opens in a new window regardless? 
```suggestion
      :href="url"
      target="_blank"
```

Reply:
Added